submissions/andysmokk/hooli-style-popup/index.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./dist/css/styles.min.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Lemonada:wght@300;400;500;600;700&display=swap"
+        rel="stylesheet" />
+    <title>POPUP</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="container">
+            <div class="header-flex-box">
+                <a class="logo-link logo-flex" href="./index.html">
+                    <div class="logo-box">
+                        <svg class="logo-img" width="40" height="40" aria-label="paw">
```

Comment:
Better move `aria-label` to the link, because the focus in the browser firstly goes to the link not to the image)

Reply:
Thank you. Done.